Домашнее задание #6 (RecyclerView)

hivislav opened the pull request:
//Реализован код для списка заметок
//Создана активити для редактирования данных с помощью CardView
//реализован механизм заполнения заметки и сохранения ее


** не смог разобраться как реализовать именно редактирование конкретной заметки, а не заполнения ее с нуля (проблема видимо в способе "сборки" самой заметки в EditNote, но решение найти пока не смог)

hivislav committed:
Домашнее задание #6 (RecyclerView)

//Реализован код для списка заметок
//Создана активити для редактирования данных с помощью CardView
//реализован механизм заполнения заметки и сохранения ее

** не смог разобраться как реализовать именно редактирование конкретной заметки, а не заполнения ее с нуля (проблема видимо в способе "сборки" самой заметки в EditNote, но решение найти пока не смог)

hivislav commented:
![Screenshot_20220305-170327_Note App Homework](https://user-images.githubusercontent.com/99187338/156883289-b86f58c5-eefb-480d-88ea-55fcc139d85b.jpg)
![Screenshot_20220305-170423_Note App Homework](https://user-images.githubusercontent.com/99187338/156883291-c57b0ef4-e073-41a5-b341-a4ad1919fcda.jpg)
![Screenshot_20220305-170431_Note App Homework](https://user-images.githubusercontent.com/99187338/156883293-1ecf7eca-b18e-47a9-b87f-f1a5fd9686c1.jpg)
![Screenshot_20220305-170320_Note App Homework](https://user-images.githubusercontent.com/99187338/156883296-2e54c632-14cb-4b7f-aadd-2f24e02ea0cd.jpg)

hivislav committed:
исправил styles.xml, убрав лишние кавычки, которые сломали визуал карточки

carnivoire123 commented on app/src/main/res/layout/activity_edit_note.xml:
```diff
@@ -0,0 +1,40 @@
+                android:layout_gravity="center_horizontal"
+                android:layout_width="100dp"
+                android:layout_height="50dp"
+                android:layout_marginVertical="600dp"
```
За это могут уволить - никогда не используйте в лэйаутах размеры - на экранах меньшей высоты все улетит за нижнюю грань

hivislav committed:
//Исправлен механизм редактирования заметки, теперь все работает корректно
//Из лэйаутов убраны абсолютные значения - размер теперь привязан к весу через Linear
//некоторые ранее пропущенные значения убраны в ресурсы

hivislav committed:
//Исправлен механизм редактирования заметки, теперь все работает корректно
//Из лэйаутов убраны абсолютные значения - размер теперь привязан к весу через Linear
//некоторые ранее пропущенные значения убраны в ресурсы
